index.html

```diff
@@ -63,5 +63,143 @@ <h2 class="let">LET'S CONNECT</h2>
       
     </div>
     </section>
+    <section class="works">
+      <div class="card">
+        <div class="card__image">
+          <img src="images/tox1.png" alt="">
+        </div>
+        <div class="card__content">
+          <h2 class="card__content--heading">Tonic</h2>
+          <ul class="card__content--item1">
+            <li class="card__item--heading"> Canopy</li>
+            <li> <img src="images/Counter.png" alt="Counter"> Back End Dev</li>
+            <li> <img src="images/Counter.png" alt="Counter"> 2015</li>
+          </ul>
+          <p class="card__content--desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="card__content--item2">
+            <li class="tech__btn">html</li>
+            <li class="tech__btn">css</li>
+            <li class="tech__btn">Javascript</li>
+          </ul>
+          <button class="btn">See Project</button>
+        </div>
+      </div>
+      
+      <div class="card">
+        <div class="card__image">
+          <img src="images/mul2.png" alt="mul2.png">
+        </div>
+        <div class="card__content">
+          <h2 class="card__content--heading">Multi-Post Stories</h2>
+          <ul class="card__content--item1">
+            <li class="card__item--heading"> Canopy</li>
+            <li> <img src="images/Counter.png" alt="Counter"> Back End Dev</li>
+            <li> <img src="images/Counter.png" alt="Counter"> 2015</li>
+          </ul>
+          <p class="card__content--desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <ul class="card__content--item2">
+            <li class="tech__btn">html</li>
+            <li class="tech__btn">css</li>
+            <li class="tech__btn">Javascript</li>
+          </ul>
+          <button class="btn">See Project</button>
+        </div>
+      </div>
+      <div class="card">
+        <div class="card__image">
+          <img src="images/ton21.png" alt="ton21">
```

Comment:
- [ ] Speaking of specifying paths for images, here is another bug that is preventing your beautiful images to be displayed on your page 🤔. To fix this kindly capitalize the first letter of the `images` directory, so you end up with something like this `src="Images/tox1.png"`

![image](https://user-images.githubusercontent.com/43172164/206504012-413e61cf-9bef-456b-ab8b-0ff5d625d5aa.png)